src/providers/delimitedtext/qgsdelimitedtextfeatureiterator.cpp

```diff
@@ -104,6 +104,14 @@ QgsDelimitedTextFeatureIterator::QgsDelimitedTextFeatureIterator( QgsDelimitedTe
     mMode = FeatureIds;
     mTestSubset = false;
   }
+  else if ( request.filterType() == QgsFeatureRequest::FilterFids )
+  {
+    QgsDebugMsg( QStringLiteral( "Configuring for returning list of ids" ) );
```

Comment:
Is that something we want to keep for prod?

Reply:
Not sure to understand the question. We need to support the fids filter for tests and prod, no?

Reply:
It was only about the debug messge

Reply:
I removed this part anyway